static/templates/recipient_row.hbs

```diff
@@ -50,6 +50,12 @@
                     <span class="topic_edit_form" id="{{id}}"></span>
                 </span>
 
+                {{#if topic_is_resolved}}
+                    <i class="fa fa-check on_hover_topic_unresolve recipient_bar_icon" title="{{t 'Unresolve topic' }}" role="button" tabindex="0" aria-label="{{t 'Unresolve topic' }}"></i>
+                {{else}}
+                    <i class="fa fa-check on_hover_topic_resolve recipient_bar_icon" title="{{t 'Resolve topic' }}" role="button" tabindex="0" aria-label="{{t 'Resolve topic' }}"></i>
+                {{/if}}
+
```

Comment:
After playing with this, I think we want to migrate all of the `recipient_bar_icon` elements to use TippyJS tooltips for consistency with the rest of the message view UI.  Feel free to do that in a follow-up PR or extra commits in this PR series.

Reply:
Also, the labels should match what we use in the popover: "Mark topic resolved" (or whatever it was).  It's very important to use consistent terminology for a feature in user-facing UI.